#camera information
sensor_msgs/CameraInfo camera_info

#3D-coordinates of the real world pattern that is detected in its own coordinates
geometry_msgs/Point[] pattern_coordinates